src/common/ox_tox.rs

```diff
@@ -0,0 +1,155 @@
+use crate::common::CNS_COEFFICIENTS;
+use crate::{Minutes, Pressure, Seconds, StepData};
+
+use super::{CNSCoeffRow, CNSPercent, MbarPressure};
+
+const CNS_ELIMINATION_HALF_TIME_MINUTES: Minutes = 90;
+const CNS_LIMIT_OVER_MAX_PP02: Seconds = 400;
+
+#[derive(Copy, Clone, Debug)]
+#[derive(PartialEq)]
+pub struct OxTox {
+    cns: CNSPercent
+}
+
+impl Default for OxTox {
+    fn default() -> Self {
+        Self {
+            cns: 0.
+        }
+    }
+}
+
+impl OxTox {
+    pub fn cns(&self) -> CNSPercent {
+        self.cns
+    }
```

Comment:
A to w ogóle ma sens? bo jak już masz ten 0xTox to przecież możesz zrobić .cns nie musisz wołać metody?